### PR TITLE
Add grid dots to Ecosystem section

### PR DESCRIPTION
**Pull request by jeffdaley**
Partially addresses #550 

Adds (new) `<GridDots />` component to the Ember Ecosystem section of the homepage.

CleanShot 2020-12-11 at 16 40 20@2x

The SVG itself looks like this: 
CleanShot 2020-12-11 at 16 40 49@2x

### CODE REVIEW

File: app/styles/components/ecosystem-icons.css [hunk 1]
```diff
@@ -1,3 +1,13 @@
+.ecosystem-icons-container {
+  position: relative;
+}
+
+.ecosystem-icons-container .grid-dots {
+  position: absolute;
+  top: -40px;
+  right: 0px;
```
Comment (ijlee2) [inline app/styles/components/ecosystem-icons.css:8]: Looks like `stylelint` is asking for `right: 0` instead.

```suggestion
  right: 0;
```

**Comment by ijlee2** [comment]
Hi, @jeffdaley. Thanks for working on creating `<GridDots>` component.

I pushed a commit to fix the linting error. I was wondering if there's someone specific (maybe from the website team?) whom you worked with/you'd like for review?

**Comment by jeffdaley** [comment]
@ijlee2 Thanks for committing that fix. I don't have anyone specific in mind, but someone from the website team makes sense to review.

**Comment by ijlee2** [comment]
@MelSumner or @mansona Could one of you review this?

**Review by ijlee2 — APPROVED** [review]
@jeffdaley Apologies for a delay in the review. I checked the Percy snapshots and the deploy preview, and think we can merge this PR.

Thank you for your help! ✨

**Pull request merged**